journal_brief/cli/main.py

```diff
@@ -200,6 +208,45 @@ def get_formatters(self):
 
         return formatters
 
+    def send_email(self, output):
+        email = self.config.get('email')
+
+        if len(output) == 0:
```

Comment:
Minor nit: could use `if not output:` here.

Reply:
Done.